src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -93,6 +95,24 @@ static const TypeMapping dnnlConvolutionTypeMapping {
     // @todo should we fallback to _fxx instead of _f32 (currenly legacy logic is replicated)
     {{_any, _any, _any, _any},                     pt(just<f32>(), just<f32>(), just<f32>(), just<f32>())},
 };
+
+static const TypeMapping dnnlMatMulTypeMapping {
```

Comment:
Have you alligned this mapping with supported configurations on oneDNN side https://github.com/openvinotoolkit/oneDNN/blob/6b99866a4531e38a74d1de36d5b366c54c5e6cc3/src/cpu/x64/matmul/brgemm_matmul_utils.cpp#L186-L201? Seems like some combinations are not supported by the kernel

Reply:
I tried to align with this info:
https://oneapi-src.github.io/oneDNN/dev_guide_matmul.html#data-types
and the current fullyconnected use cases
Could you please provide an example of missed combination?